docs/dev/api/jobs.md

```diff
@@ -274,6 +274,10 @@ curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
 
 Edit job attributes based on parameters passed in the request, including setting the status of the job. Any parameter for which a new value is provided in the request will replace the existing value. For example, if you provide a set of tags, they will not be added to the current tags; they will replace them, so make sure you pass the entire set you wish to assign.
 
+:::note RDC Jobs
+This call is specific to jobs running in simulation. To update a job for real devices, use the [Selenium JavaScript Executor](/basics/test-config-annotation/test-annotation/#selenium-javascript-executor).
```

Comment:
It's not only the update job status, also the update job name. Can we also mention that here?